### ISSUE TITLE
only works on last added element

### ISSUE THREAD

**Issue by iwebnaut** [iFgR/vue-shortkey#24]
Seems that only the last element I've added reacts to the shortcut. I added 2 elements first and tested it and only second element reacted to the shortcut, then I added the third element and only that element reacted to the shortcut.

Is this the way it should be or am I doing something strange with my components :) 
I am using this inside file component which I then use to create elements with v-for. 

I am also using vuex, not sure if that could somehow interfere with shortcut directive?

Here is how i am using it:

    <div v-shortkey.once="['ctrl', 'alt', 'o']" @shortkey="showHeader=!showHeader" v-show="showHeader" class="content-block-header">

Basically, I want to hide the header div of every component that I've added to the list and that is being displayed on the page.

**Comment by francois2metz** [comment]
@iwebnaut Could you please give us a reproductible test case, using http://jsfiddle.net/ for instance?

**Comment by francois2metz** [comment]
Closing as there is no details.

**Comment by enricribas** [comment]
Add two v-shortkey maps on a page, only one will work.
